portal/2022-10-18-changelog.md

```diff
@@ -0,0 +1,8 @@
+---
+authors: posdealer
+slug: portal/bugfix-receipt-calculation
+tags: [Portal,BugFix]
+---
+
+# Fix for receipt calculation
+We have fixed receipt calculation when it contains negative quantity or amount. Total amount is summed up correctly now for all receipts.
```

Comment:
Maybe we can emphasize here that this is only relevant for the "receipt detail view" in the Portal, to not risk that people confuse this with something happening in the Middleware. And maybe that only the visualization was affected, but all data is of course stored correctly. 😅

Reply:
Make sense, I extended the text.